.github/workflows/unittests.yml

```diff
@@ -96,12 +96,11 @@ jobs:
           mvn clean install -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -P arrow-jni -am -Darrow.cpp.build.dir=/tmp/arrow/cpp/build/release/ -DskipTests -Dcheckstyle.skip
       - name: Run unit tests
         run: |
-          sed -i "199, 202d" pom.xml
           mvn clean install -N
           cd arrow-data-source
           mvn clean install -DskipTests -Dbuild_arrow=OFF
           cd ..
-          mvn clean package -P full-scala-compiler -Phadoop-2.7.4 -am -pl native-sql-engine/core -DskipTests -Dbuild_arrow=OFF
```

Comment:
-Phadoop-2.7.4 is required since the default Hadoop version has been changed to 3.2.0.

Reply:
Just reverted the changes. Thanks!